magento/magento2
# Resolve Admin panel is not accessible after limited permissions set to at least one admin account issue25881

### Description (*)


1.  Resolve magento/magento2#25881: Admin panel is not accessible after limited permissions set to at least one admin account

### Fixed Issues (if relevant)

1. magento/magento2#25881: Admin panel is not accessible after limited permissions set to at least one admin account

### Manual testing scenarios (*)

- Login to admin panel
- Go to System->User Roles
- Create a new role:
- In Role Resources check only Media Gallery
![image](https://user-images.githubusercontent.com/2028541/69998457-3e97f580-154e-11ea-85d7-7d8efd8ecfbc.png)
- Create a new admin user account and assign the created role to it
- Log out from admin panel
- Log in to admin panel as a newly created user
- Log in to admin panel as an admin having full permissions

#### Expected result
The new admin user is logged into the admin panel.
Other admin accounts are not affected.

### Questions or comments


### Contribution checklist (*)
